nixos/modules/programs/browserpass.nix

```diff
@@ -6,21 +6,17 @@ with lib;
 
   ###### interface
   options = {
-    programs.browserpass = {
-      enable = mkOption {
-        default = false;
-        type = types.bool;
-        description = ''
-          Whether to install the NativeMessaging configuration for installed browsers.
-        '';
-      };
-    };
+    programs.browserpass.enable = mkEnableOption "the NativeMessaging configuration for installed browsers.";
```

Comment:
@dermetfan unfortunately it doesn't work because Firefox expects the path to the native messaging hosts to be `/usr/{lib,lib64,share}/mozilla/native-messaging-hosts/<name>.json`, see https://wiki.mozilla.org/WebExtensions/Native_Messaging

As far as I can tell the only solution is to patch Firefox to use an environment variable (like `MOZ_PLUGIN_PATH`) or to wrap it in `buildFHSUserEnv`.